documentation/src/main/asciidoc/topics/con_cache_store_sql.adoc

```diff
@@ -1,17 +1,16 @@
 [id='sql-cache-store_{context}']
 = SQL cache stores
 
-SQL cache stores pre-load {brandname} caches from databases, with a supported dialect, through JDBC connections.
-
-You can create two types of SQL cache store:
+SQL cache stores let you load {brandname} caches from database tables.
+{brandname} offers two types of SQL cache store:
 
 Table:: {brandname} loads entries from a single database table.
-Query:: {brandname} uses SQL queries to load entries as well as perform insert, update, and delete operations.
+Query:: {brandname} uses SQL queries to load entries from multiple database tables, select data from individual columns in database table rows, or perform insert, update, and delete operations.
 
 Both SQL table and query stores:
 
 * Allow read and write operations to persistent storage.
-* Can be read-only and act as a Cache Loader.
+* Can be read-only and act as a cache loader.
```

Comment:
I noticed we don't document how this is done anywhere in the documentation. Maybe we should have a read only and write only example somewhere as this store especially will be a prime target for the former.

Reply:
I'll create another Jira for this. I'd rather expand on the documentation in a separate effort. https://issues.redhat.com/browse/ISPN-13579